setup.cfg

```
@@ -50,7 +50,7 @@ image =
   scikit-image>=0.16.0
   scipy>=1.3.0
 jpeg2000 =
-  glymur>=0.8.18,!=0.9.0
+  glymur>=0.8.18,!=0.9.0,<0.9.5
```

hmm, we could do this:
```suggestion
  glymur>=0.8.18,!=0.9.0,!=0.9.5
```
and hope that it's fixed in the next release? (Which I would say is probable).